WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -602,12 +601,13 @@ - (void)configureAnalytics
 #pragma mark - App Rating
 
 - (void)initializeAppRatingUtility
-{    
+{
+    AppRatingUtility *utility = [AppRatingUtility shared];
     NSString *version = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
-    [AppRatingUtility registerSection:@"notifications" withSignificantEventCount:5];
-    [AppRatingUtility setSystemWideSignificantEventsCount:10];
-    [AppRatingUtility initializeForVersion:version];
-    [AppRatingUtility checkIfAppReviewPromptsHaveBeenDisabled:nil failure:^{
+    [utility registerSection:@"notifications" withSignificantEventCount:5];
```

Comment:
I was going to add this but I'm not sure where to put it. Besides, I'm not a big fan of the "every string must be a constant" approach. In this particular case, we have assertions in place, so if there's a typo in the section name, you'd get a crash on debug.